### DISCUSSION TITLE
Quick fixes

### DISCUSSION THREAD

**Discussion by erennatala** [erennatala/CSE3063F21P1_GRP5#73] [General]
1.  Collision error raiseError()
2.  Registrator do not add same elective to basket
3.  Make Error calls on student addError()
4.  Make proper calls to create and add transcript attributes
5.  Transcript Writer
6.  gpa cgpa calculation

**Comment by erennatala** [comment]
prerequsites json changed take that from @GranFenrir

**Comment by erennatala** [comment]
Lab courses may be deleted

**Comment by erennatala** [comment]
department output
logger

**Comment by erennatala** [comment]
instructor collision check will be adjusted
transferring next semester will be implemented after grading

**Comment by erennatala** [comment]
instructor collision Nontaken type adjustment

**Comment by erennatala** [comment]
Courses are not adding to Nontaken List at Approver class when an error occur
Config parameter season
Season Courses
Cgpa ,GPA
Comment , Logger
Transcript Reader
Nontaken Courses will be written and read at transcript

**Reply by keremmican** [reply]
config read done

**Reply by keremmican** [reply]
cgpa, gpa and writing them to transcriptwriter is done

**Comment by erennatala** [comment]
When next semester : error list will be cleared from student
